src/gui/MainWindow.cpp

```diff
@@ -2046,8 +2045,8 @@ void MainWindow::RecreateMenu()
 
 	if (!m_game_launched)
 	{
-		m_loadMenuItem = m_fileMenu->Append(MAINFRAME_MENU_ID_FILE_LOAD, _("&Load"));
-		m_installUpdateMenuItem = m_fileMenu->Append(MAINFRAME_MENU_ID_FILE_INSTALL_UPDATE, _("&Install game title, update or DLC"));
+		m_loadMenuItem = m_fileMenu->Append(MAINFRAME_MENU_ID_FILE_LOAD, _("&Load..."));
+		m_installUpdateMenuItem = m_fileMenu->Append(MAINFRAME_MENU_ID_FILE_INSTALL_UPDATE, _("&Install game title, update or DLC..."));
```

Comment:
The reason for adding ellipsis here is because those actions require additional information (input or a selection) to carry out the operation, see https://stackoverflow.com/questions/637683/when-to-use-ellipsis-after-menu-items/637708#637708